src/assets/translations/en/main.json

```diff
@@ -590,7 +590,29 @@
     "buyCryptoShort": "Buy/Sell",
     "arkeo": "Arkeo Airdrop",
     "lending": "Lending",
-    "pools": "Pools"
+    "pools": "Pools",
+    "actionMenu": {
+      "send": {
+        "title": "Send",
+        "body": "Send your crypto"
+      },
+      "receive": {
+        "title": "Receive",
+        "body": "Receive crypto"
+      },
+      "trade": {
+        "title": "Trade or Bridge",
+        "body": "Trade of bridge your crypto"
+      },
+      "buy": {
+        "title": "Buy or Sell",
+        "body": "Buy or sell crypto with credit card"
```

Comment:
Technically not only as some fiat ramp providers may just provide other options, but I think that's a nice oversimplification!